resource_processor/run.sh

```diff
@@ -28,7 +28,7 @@ name = "aad_auth"
 plugin = "azure.keyvault"
 
 [secrets.config]
-vault = "${KEY_VAULT_NAME}"
+vault-url = "${KEY_VAULT_URL}"
```

Comment:
Since this is for porter v0 with an older plugin, I don't think you should change it.
porter v1 is using the config.yaml defined as a file.

Reply:
I reverted it to the same way that it was(the comment is up there)